vision_datasets/common/data_manifest.py

```diff
@@ -324,6 +324,42 @@ def sample_subset_by_ratio(self, sampling_ratio):
         sampled_images = [self.images[i] for i in sampled_image_ids]
         return DatasetManifest(sampled_images, self.labelmap, self.data_type)
 
+    def sample_few_shots_subset_greedy(self, n_shots, random_seed=0):
+        """Greedy few-shots sampling method.
+        Randomly pick images from the original datasets until all classes have at least n_shots images.
```

Comment:
can we update the original sample_few_shots_subset docstring to emphasize that one is aiming for balanced sampling

One other thing is that the original few_shot method is greedy as well, I am thinking we can probably re-use the previous interface but having a flavor ['balanced', 'guaranteed'] or something, underlying functions can be two separated sampling method _few_shot_sampling_balanced, _few_shot_sampling_ganranteed or something

Reply:
Might as well mention the expected outcome of negative images as well

Reply:
I've updated the comments and mentioned negative images. Also, changed the parameter name from "n_shots" to "num_min_images_per_class" in order to emphasize the differences.

The two methods have different definition of the parameter, so I think it's safer to keep different interfaces.